javascript-data-model/order-history.js

```diff
@@ -0,0 +1,32 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4th, 2020',
+    total: '$34',
+    products: [{ name: 'JavaScript for impatient programmers' }],
+    author: 'Rauschmayer, Dr.Axel',
+    shipTo: 'JS Masher',
+    images: [{ image1: 'url' }]
```

Comment:
Instead of having separate arrays for products and images, you should group these up to an array of the items bought, and have an object for each item with all the data about that item.